modules/administration/pages/icinga.adoc

```diff
@@ -151,7 +160,7 @@ systemctl enable apache2.service
 systemctl start apache2.service
 ----
 
-. Login to the WebGUI at: http://localhost/icinga.
+. Login to the WebGUI at: http://<client-hostname>/icinga.
```

Comment:
If we say 'localhost', the user needs to use a browser on the very same client. I think it makes more sense to use the hostname for the client.

Reply:
This should probably just be *hostname*